.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: Integration Tests
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 4 * * *"
+  push:
+    tags:
+      - "*"
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  ALLOW_PLOTTING: true
+  SHELLOPTS: 'errexit:pipefail'
+
+jobs:
+  # For now this is just MNE-Python, but others could be added
+  mne:
+    name: MNE-Python
+    runs-on: ubuntu-22.04
+    env:
+      DISPLAY: ':99.0'
+      MNE_LOGGING_LEVEL: 'info'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: git clone --depth=1 https://github.com/mne-tools/mne-python.git --branch main --single-branch
+      - run: ./mne-python/tools/setup_xvfb.sh
```

Comment:
@larsoner, any reason you're using this and not: https://github.com/pyvista/setup-headless-display-action

Reply:
Our `setup_xvfb.sh` is a little bit of a misnomer because it actually also takes care of any `xcb` dependencies that the various PyQt / PySides might need to work properly. If `setup-headless-display-action` also does this, then yes this can be replaced